example/hyprland.conf

```
@@ -27,7 +27,7 @@ general {
     col.active_border=0x66ee1111
     col.inactive_border=0x66333333
 
-    apply_sens_to_raw=0 # do not apply the sensitivity to raw input (e.g. used by games where you aim)
+    apply_sens_to_raw=0 # apply the sensitivity to raw input (e.g. used by games where you aim using your mouse)
```

```suggestion
    apply_sens_to_raw=0 # whether to apply the sensitivity to raw input (e.g. used by games where you aim using your mouse)
```
this seems more descriptive to me

---

yup seems more descriptive to me as well